### ISSUE TITLE
#bc-ignore! for NSLocalizedString

### ISSUE THREAD

**Issue by kafejo** [Flinesoft/BartyCrouch#15]
Is it possible to exclude NSLocalizedStrings that have `#bc-ignore!` in comment? The use case is that when we use .stringsdict to handle pluralization, we don't need to add these keys into Localizable.strings file.

Example
```
String.localizedStringWithFormat(NSLocalizedString("%d minute(s) ago", comment: "x minute(s) ago"), components.minute)
```
The `%d minute(s) ago` key will be taken from Localizable.stringsdict file, not from Localizable.strings.

Another possible solution is to scan Localizable.stringsdict file for keys and exclude them from Localizable.strings file.

**Comment by kafejo** [comment]
see #16 pull request

**Comment by Dschee** [comment]
This was implemented in version 3.1.0. Thanks for reporting!
